packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/evaluate-expression.ts

```diff
@@ -22,6 +22,13 @@ export interface EvaluateExpressionProps extends sfn.TaskStateBaseProps {
    * @default lambda.Runtime.NODEJS_18_X
    */
   readonly runtime?: lambda.Runtime;
+
+  /**
+   * This is the role that will be assumed by the function upon execution.
+   *
+   * @default - a role will be automatically created
```

Comment:
nit: could you remove the hyphen to match the other `@default`.

Reply:
https://github.com/aws/aws-cdk/pull/29288#discussion_r1580277065